docs/logo/fizzy_logo_turquoise.svg

```diff
@@ -2,25 +2,30 @@
 <!-- Generator: Adobe Illustrator 24.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
 	 viewBox="0 0 600 600" style="enable-background:new 0 0 600 600;" xml:space="preserve">
+<title>Fizzy logo (turquoise)</title>
 <style type="text/css">
-	.st0{fill:#53BAD8;}
-	.st1{fill:#EDC433;}
-	.st2{fill:#FFFFFF;}
+	.background{fill:#53BAD8;}
+	.yellow{fill:#EDC433;}
+	.white{fill:#FFFFFF;}
 </style>
-<g id="Layer_2_1_">
-	<rect x="-18.5" y="-11.5" class="st0" width="631.6" height="623.3"/>
+<!-- Background -->
```

Comment:
I guess you can just change `id="Background"` instead of a comment.

Reply:
I know, considered that too, not sure why I didn't do that in the end.

Reply:
Maybe because I wasn't sure what to name `Layer_1` :wink: